helm/chart-operator/templates/deployment.yaml

```diff
@@ -70,7 +70,11 @@ spec:
           value: "5"
       initContainers:
       - name: wait-for-quay
+        {{ if .Values.isManagementCluster }}
+        image: "{{ .Values.registry.domain }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        {{ else }}
         image: "{{ .Values.image.registry }}/{{ .Values.image.name }}:{{ .Values.image.tag }}"
+        {{ end }}
```

Comment:
For my interest: why different images on MC and WC?

Reply:
I don't know, but also don't want to investigate this now. Can be easily changed via config repo later

Reply:
We use `image.registry` in the default catalog. So I'd prefer to use that and remove some of the conditionals.

But I'm fine if we do that in batman once this migration is done.

https://github.com/giantswarm/installations/blob/80e965ba6c529a52f491932fb0972e851c29b374/giraffe/appcatalog/default-appcatalog-values.yaml#L10-L11